### PR TITLE
added 'id' attribute to html in various places to ease selenium testing

### PR DESCRIPTION
**Pull request by mswertz**
Towards automated testing I have been testing PageObject design pattern. See first examples at https://github.com/mswertz/molgenis-selenium-test

To ease the life of the selenium maintainer it helps to mark key elements in the user flow using an identifier, which is done in this branch.

### CODE REVIEW

File: molgenis-core-ui/src/main/resources/js/component/AlertMessage.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
 						span({'aria-hidden': true,}, String.fromCharCode(215)) // &times;
 					) : null,
 					this.props.type === 'danger' ? molgenis.ui.Icon({name: 'exclamation-sign'}) : null,
-					this.props.type === 'danger' ? ' ' + this.props.message : this.props.message  
+					span({id: 'molgenis-alert'},this.props.type === 'danger' ? ' ' + this.props.message : this.props.message)
```
Comment (dennishendriksen) [inline molgenis-core-ui/src/main/resources/js/component/AlertMessage.js]: Multiple alerts can be displayed at the same time, element ids must be unique.

File: molgenis-core-ui/src/main/resources/js/molgenis.js [hunk 2]
```diff
@@ -30,9 +30,11 @@
 		items.push(type.charAt(0).toUpperCase() + type.slice(1));
 		items.push('!</strong> ');
 		$.each(alerts, function(i, alert) {
+		    items.push('<span id="molgenis-alert">');
```
Comment (dennishendriksen) [inline molgenis-core-ui/src/main/resources/js/molgenis.js]: Multiple alerts can be displayed at the same time, element ids must be unique.

File: molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl [hunk 3]
```diff
@@ -151,7 +151,7 @@
                         </#if>
                         
                         <#if messageType??>
-                            <div class="alert alert-${messageType}"><button type="button" class="close" data-dismiss="alert">&times;</button><strong>${messageType?capitalize}!</strong> ${message?html}</div>
+                            <div class="alert alert-${messageType}"><button type="button" class="close" data-dismiss="alert">&times;</button><strong>${messageType?capitalize}!</strong><span id="molgenis-alert" ${message?html}</div>
```
Comment (dennishendriksen) [inline molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl]: Multiple alerts can be displayed at the same time, element ids must be unique.

Comment (dennishendriksen) [inline molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl]: Duplicate ids when this alert is shown in combination with plugin specific alerts.

File: molgenis-data-import/src/main/resources/templates/ImportResultsWizardPage.ftl [hunk 4]
```diff
@@ -158,7 +158,7 @@
                 error: function (xhr, textStatus, errorThrown) {
                     var errorMessage = JSON.parse(xhr.responseText).errorMessage;
                     $('#plugin-container .alert').remove();
-                    $('#plugin-container').prepend('<div class="alert alert-danger"><button type="button" class="close" data-dismiss="alert">&times;</button><strong>Error!</strong> ' + errorMessage + '</div>');
+                    $('#plugin-container').prepend('<div class="alert alert-danger"><button type="button" class="close" data-dismiss="alert">&times;</button><strong>Error!</strong><span id="molgenis-alert"> ' + errorMessage + '</span></div>');
```
Comment (dennishendriksen) [inline molgenis-data-import/src/main/resources/templates/ImportResultsWizardPage.ftl]: Duplicate ids when this alert is shown in combination molgenis-header alert.

File: molgenis-data-import/src/main/resources/templates/ImportResultsWizardPage.ftl [hunk 5]
```diff
@@ -175,7 +175,7 @@
...
```
Comment (dennishendriksen) [inline molgenis-data-import/src/main/resources/templates/ImportResultsWizardPage.ftl]: Duplicate ids when this alert is shown in combination molgenis-header alert.

File: molgenis-data-import/src/main/resources/templates/ValidationResultWizardPage.ftl [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <#if wizard.validationMessage??>
 	<script>
 		$(function() {
-			$('.bwizard-steps').after($('<div class="alert alert-block alert-danger">${wizard.validationMessage?js_string}</div>'));
+			$('.bwizard-steps').after($('<div class="alert alert-block alert-danger"><span id="molgenis-alert">${wizard.validationMessage?js_string}</span></div>'));
```
Comment (dennishendriksen) [inline molgenis-data-import/src/main/resources/templates/ValidationResultWizardPage.ftl]: Duplicate ids when this alert is shown in combination molgenis-header alert.

**Comment by fdlk** [comment]
For the alerts, By.cssSelector should work fine too, I think?

**Pull request closed without merging**